options/posix/include/mlibc/posix-sysdeps.hpp

```diff
@@ -157,6 +157,8 @@ int sys_vm_unmap(void *pointer, size_t size);
 [[gnu::weak]] int sys_before_cancellable_syscall(ucontext_t *uctx);
 [[gnu::weak]] int sys_tgkill(int tgid, int tid, int sig);
 
+[[gnu::weak]] int sys_fchownat(int dirfd, const char *pathname, uid_t owner, gid_t group, int flags, bool dereference);
```

Comment:
Why do we have `bool dereference`? Following symlinks is just `AT_SYMLINK_NOFOLLOW` in flags, isn't it? See https://man7.org/linux/man-pages/man2/lchown.2.html

Reply:
I didn't think of that, it was to distinguish `chown` and `lchown`. I'll change it to AT_SYMLINK_NOFOLLOW